worker-sandbox/src/lib.rs

```diff
@@ -119,6 +119,28 @@ pub async fn main(req: Request, env: Env, _ctx: worker::Context) -> Result<Respo
             // Let the integration tests have some way of knowing if we successfully received the closed event.
             Response::ok(got_close_event)
         })
+        .get_async("/ws-client", |_, _| async move {
+            let ws = worker::connect("wss://echo.zeb.workers.dev/".parse()?).await?;
```

Comment:
is `worker` the top-level crate? if so, I think we could be more explicit here and instead add this as a method to `WebSocket` maybe? adding a public `connect` method at the current level could be misconstrued with many other "connect"-like APIs, i.e. HTTP CONNECT (which we don't support), some future TCP socket API, etc.. 

was there an issue with associating this method with one of the WebSocket types?

Reply:
Workers is the top-level crate, the reason I did this was to try to follow the API of other WebSocket libraries as close as possible. I agree that the name `connect` is vague, I think either a method like `WebSocket::connect` or a more descriptive path to the function like `worker::ws::connect`  would be better. I'll go ahead with `WebSocket::connect` if that's your preference.